src/Solutions/AddMissingPermissionsTraitSolution.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Junges\ACL\Solutions;
+
+use Facade\IgnitionContracts\RunnableSolution;
+use Junges\ACL\Solutions\Utilities\InsertMissingTrait;
+
+class AddMissingPermissionsTraitSolution implements RunnableSolution
+{
+    /**
+     * The class which the method is called from.
+     * 
+     * @var string
+     */
+    private $class;
+
+    /**
+     * Construct new Solution.
+     */
+    public function __construct(string $class = null)
+    {
+        if ($class) {
+            $this->class = explode('::', $class)[0];
+        }
+    }
+
+    public function getSolutionTitle(): string
+    {
+        return "You forgot to use PermissionsTrait on `{$this->class}`";
+    }
+
+    public function getSolutionDescription(): string
+    {
+        return "You can add the missing trait yourself by putting the `use PermissionsTrait` on you `{$this->class}`, or run the below solution.";
```

Comment:
I think it should be `on your`, instead of `on you`.

Reply:
Thank you 👍